pepdbagent/db_utils.py

```diff
@@ -133,10 +127,22 @@ class Samples(Base):
 
     id: Mapped[int] = mapped_column(primary_key=True)
     sample: Mapped[dict] = mapped_column(JSON, server_default=FetchedValue())
-    row_number: Mapped[int]
+    row_number: Mapped[int]  # TODO: should be removed
```

Comment:
Do we need to remove this now?

Reply:
yes, but in the future, after database will be done